pybaseball/playerid_lookup.py

```diff
@@ -3,12 +3,13 @@
 import io
 import os
 
-# dropped key_uuid. looks like a has we wouldn't need for anything.
+# Dropped key_uuid. Looks like a has we wouldn't need it for anything.
```

Comment:
Way to fix the second mistake here, though I don't know what the first was supposed to be to begin with. I have no clue at all what this comment means, tbh